doc/tutorials/02-charged_system/scripts/nacl_units_confined_vis.py

```diff
@@ -44,7 +44,7 @@
 #l_bjerrum = 0.885^2 * e^2/(4*pi*epsilon_0*k_B*T)
 l_bjerrum = 130878.0 / temp
 Vz = 0  # potential difference between the electrodes
-Ez = 364.5  # conversion from potential to electrical field
+Vz_to_Ez = 364.5  # conversion from potential to electrical field
```

Comment:
I think we should use capital letter names for all constants https://www.python.org/dev/peps/pep-0008/#id48

Reply:
Deferred to espresso 5, moved to a dedicated issue.